content/recipes/migration.mdx

```diff
@@ -17,11 +17,11 @@ The library has been internally rewritten to use new Context. You will need Reac
 
 ### What about mobx-react 5.x?
 
-If you are forced to stick to the older version (most likely because older React), there isn't really anything you can do right now and just keep using inject as you are used to.
+If you are forced to stick to the older version (most likely because oof lder React), there isn't really anything you can do right now and just keep using inject as you are used to.
```

Comment:
Second honest typo? :) Not sure how you be fixing something and making it actually worse :)

Reply:
Oh man, this is not good! Fixed now.